content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: OpenTelemetry for Generative AI
+linkTitle: OpenTelemetry for Generative AI
+date: 2024-11-09
+author: >-
+  [Drew Robbins](https://github.com/drewby) (Microsoft),  [Liudmila
+  Molkova](https://github.com/lmolkova) (Microsoft)
+---
```

Comment:
```suggestion
issue: 
sig: SIG GenAI Observability
---
```

@drewby can you create an issue with the same comment as you have here in the PR description, so we can link it to this blog post. Since this PR has everything needed (SIG + sponsors) we can keep it, but we want to establish this practice so let's set a good example

Reply:
will do